rpcs3/Emu/Memory/vm.cpp

```diff
@@ -1182,7 +1182,7 @@ namespace vm
 			if (is_write)
 				std::swap(src, dst);
 
-			if (size <= 16 && std::popcount(size) == 1 && (addr & (size - 1)) == 0)
+			if (size <= 16 && (size & (size - 1)) == 0 && (addr & (size - 1)) == 0)
```

Comment:
```suggestion
			if (size <= 16 && ((size | addr) & (size - 1)) == 0)
```

Reply:
There is little point to optimize it by hand (the outcome is the same), it only hurts readability due to mixing two different checks.